sql/core/src/test/resources/sql-tests/inputs/correlated.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TEMPORARY VIEW t1 AS SELECT * FROM VALUES 1, 2 AS t1(a);
```

Comment:
Naming: lets call this `subqueries.sql`. Correlation has a different meaning in different contexts